packages/__docs__/src/Document/index.tsx

```diff
@@ -291,8 +297,105 @@ import { ${importName} } from '${esPath}'
     return doc.params || doc.returns || doc.methods || doc.props
   }
 
+  // setHeadingData() {
+  //   const { doc } = this.props
+  //   const headings = this.ref?.querySelectorAll<HTMLHeadingElement>(
+  //     'h1, h2, h3, h4, h5, h6'
+  //   )
+  //   const TOCData: DocumentState['TOCData'] = []
+  //   let hasLinkToTitle = false
+  //
+  //   headings?.forEach((h) => {
+  //     const { id, innerText, tagName } = h
+  //     const level = tagName[1]
+  //
+  //     if (['1', '2'].includes(level) || doc.id === id) {
+  //       // already has a page title
+  //       hasLinkToTitle = true
+  //     }
+  //
+  //     const data: TOCHeadingData = {
+  //       id,
+  //       innerText,
+  //       level: tagName[1]
+  //     }
+  //     TOCData.push(data)
+  //   })
+  //
+  //   if (!hasLinkToTitle) {
+  //     // if there is no link to the main title,
+  //     // add at the beginning of the list
+  //     TOCData.unshift({
+  //       id: doc.id,
+  //       innerText: doc.id,
+  //       level: '1'
+  //     })
+  //   }
+  //
+  //   if (TOCData.length === 1) {
+  //     // if there is only 1 item, clear TOC, no need for list
+  //     TOCData.pop()
+  //   }
+  //
+  //   this.setState({ TOCData })
+  // }
+  //
+  // renderTOC() {
+  //   const { doc } = this.props
+  //   const { TOCData } = this.state
+  //   const category = doc.category?.toLowerCase()
+  //
+  //   if (!TOCData || !TOCData.length) {
+  //     return null
+  //   }
+  //
+  //   const levelPaddingMap: Record<string, SpacingValues> = {
+  //     1: '0',
+  //     2: '0',
+  //     3: '0',
+  //     4: 'small',
+  //     5: 'medium',
+  //     6: 'x-large'
+  //   }
+  //
+  //   const TOC = TOCData.map((data) => {
+  //     return (
+  //       <List.Item
+  //         key={data.id}
+  //         padding={`0 0 0 ${levelPaddingMap[data.level]}`}
+  //       >
+  //         <Link href={`#${doc.id}/#${data.id}`}>{data.innerText}</Link>
+  //       </List.Item>
+  //     )
+  //   })
+  //
+  //   return (
+  //     <InstUISettingsProvider theme={instructure}>
+  //       <View as="div" margin="medium 0">
+  //         <ToggleDetails
+  //           summary="Table of Contents"
+  //           defaultExpanded={
+  //             category?.includes('components') || category === 'guides'
+  //           }
+  //           size="large"
+  //         >
+  //           <List
+  //             margin="0 0 large"
+  //             isUnstyled
+  //             size="small"
+  //             itemSpacing="xx-small"
+  //           >
+  //             {TOC}
+  //           </List>
+  //         </ToggleDetails>
+  //       </View>
+  //     </InstUISettingsProvider>
+  //   )
+  // }
```

Comment:
You can delete this :)

Reply:
Oh boy, forgot to clean up! :D Thanks :)